tests/pausing_test.go

```diff
@@ -53,136 +52,98 @@ import (
 )
 
 var _ = Describe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-compute]Pausing", decorators.SigCompute, func() {
+	const cirrosStartupTimeout = 90
```

Comment:
nit, and not introduced by this PR, but this const is better set closer to its single user.

Reply:
Updated